xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-test/xwiki-platform-user-profile-test-docker/pom.xml

```diff
@@ -44,12 +44,6 @@
       <version>${project.version}</version>
       <type>xar</type>
     </dependency>
-    <dependency>
-      <groupId>${project.groupId}</groupId>
-      <artifactId>xwiki-platform-eventstream-store</artifactId>
-      <version>${project.version}</version>
-      <scope>runtime</scope>
-    </dependency>
```

Comment:
What's the relation with the bug?

Reply:
Yes this needs to be explained and it could break the other tests needing the even stream. Did you retest all the tests too for this module?

It could mean that that eventstream store has bee added to the minimal dep list which would not be correct I think (EDIT: I checked and it doesn't seem the case). The error you were getting is out the extension being a core dep, so I'm not sure I understand the problem.

Reply:
I just re-ran the tests on xwiki-platform-user-profile with the following command, and it's passing.

```
mvn clean install -Plegacy,integration-tests,snapshot,docker -Dxwiki.checkstyle.skip=true -Dxwiki.surefire.captureconsole.skip=true -Dxwiki.revapi.skip=true
```